templates/taxi/driver_detail.html

```diff
@@ -3,11 +3,22 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
+
+    <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-secondary link-to-page">
+      Update
+    </a>
   </h1>
 
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
-  <p><strong>License number:</strong> {{ driver.license_number }}</p>
+  <p><strong>License number:</strong> {{ driver.license_number }}
+    <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn btn-outline-secondary">
+      Update driver license
+    </a>
```

Comment:
There is a redundancy in the update link for the driver's license. Consider removing this link or the one in the 'Update' button above to avoid redundancy unless there's a specific reason for having both.